src/calendar/month.jsx

```diff
@@ -64,6 +71,36 @@ export default class Month extends React.Component {
     })
   }
 
+  _pushNoticeUpdate(noticeType) {
+    const {onNoticeChange} = this.props
+    return onNoticeChange(noticeType)
+  }
+
+  _getDisabledRange(range) {
```

Comment:
We can use `areRangesOverlapping`: https://date-fns.org/docs/areRangesOverlapping